agcaballero/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,13 +19,21 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private final ArrayList<String> mssgs = {"Hello visitor!", "I am here!", "Go away."};
+
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public String doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    /** 
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello AnneMarie!</h1>");
+    */
```

Comment:
In general we avoid checking in commented-out code. Instead, I'd suggest deleting this, as it is available from source control history if you need it again in the future.

Reply:
Done